flink-python/pyflink/table/table_descriptor.py

```diff
@@ -285,6 +289,10 @@ def line_delimiter(self, delimiter):
         self._j_csv = self._j_csv.lineDelimiter(delimiter)
         return self
 
+    def schema(self, schema):
+        self._j_csv = self._j_csv.schema(schema._j_table_schema)
```

Comment:
I find that the JAVA have the follows JAVA DOC:
```
 /**
    * Sets the format schema with field names and the types. Required.
    * The table schema must not contain nested fields.
    *
    * This method overwrites existing fields added with [[field()]].
    *
    * @param schema the table schema
    */
```
It's better to align the DOC, What to you think?